addon/core/prosemirror.ts

```diff
@@ -250,14 +250,15 @@ export class ProseController {
     this.doCommand(selectAll);
     const tr = this.pm.state.tr;
     const domParser = new DOMParser();
-    tr.deleteSelection().insert(
+    tr.replaceWith(
       0,
+      tr.doc.nodeSize - 2,
       ProseParser.fromSchema(this.schema).parse(
         domParser.parseFromString(content, 'text/html')
       )
     );
+    tr.setSelection(Selection.atStart(tr.doc));
```

Comment:
the original code set the selection at the end, not a huge deal but I think it's more in line with how our other insertions work